addons/resource.images.weather.default/resources/License.txt

```diff
@@ -0,0 +1,3 @@
+XBMC Realistic Weather Icon Set by Mominur Rahman (www.illuminatedimages.co.uk) is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported License.
+
+http://creativecommons.org/licenses/by-nc-sa/3.0/
```

Comment:
Reminder:
We have permission (e-mail from author) to change the license because by-nc-sa 3.0 is not compatible with GPL. Still need to choose an appropriate one.